### PR TITLE
Fix init_client for ts and lidar mode bug

### PR DESCRIPTION
**Pull request by kairenw**
Remove stray std prefix changing ts and lidar modes to numeral strings

Addreses #437

### CODE REVIEW

**Review by bexcite — APPROVED** [review]

**Review by chrisbayruns — APPROVED** [review]

File: ouster_client/src/client.cpp [hunk 1]
```diff
@@ -300,12 +300,11 @@ std::shared_ptr<client> init_client(const std::string& hostname,
 
         // if specified (not UNSPEC), set the lidar and timestamp modes
         if (mode) {
-            sensor_http->set_config_param("lidar_mode", std::to_string(mode));
+            sensor_http->set_config_param("lidar_mode", to_string(mode));
```
Comment (Samahu) [inline ouster_client/src/client.cpp:303]:
```suggestion
            sensor_http->set_config_param("lidar_mode", sensor::to_string(lidar_mode));
```

File: ouster_client/src/client.cpp [hunk 2]
```diff
@@ -300,12 +300,11 @@ std::shared_ptr<client> init_client(const std::string& hostname,
...
         }
 
         if (ts_mode) {
-            sensor_http->set_config_param("timestamp_mode",
-                                          std::to_string(ts_mode));
+            sensor_http->set_config_param("timestamp_mode", to_string(ts_mode));
```
Comment (Samahu) [inline ouster_client/src/client.cpp:307]:
```suggestion
            sensor_http->set_config_param("timestamp_mode", sensor::to_string(ts_mode));
```

**Review by Samahu — COMMENTED** [review]
I would have preferred to replace `std::` with `sensor::`. In another PR it was pointed to me that the team would rather have fully qualified name when there is a name ambiguity.

File: ouster_client/src/client.cpp (hunk 1 above)
Comment (kairenw) [reply ouster_client/src/client.cpp:303]: :+1: we'll get it on the next one
(ditto for the other)

**Pull request merged**